plugins/modules/elastic_savedobject_info.py

```diff
@@ -36,9 +65,10 @@ def main():
         username=dict(type='str', required=True),
         password=dict(type='str', no_log=True, required=True),   
         verify_ssl_cert=dict(type='bool', default=True),
-        object_name=dict(type='str'),
-        space=dict(type='str', default='default'),
-        object_type=dict(type='str', default="default")
+        object_name=dict(type='str', required=True),
+        object_type=dict(type='str', required=True),
+        space_id=dict(type='str', default='default'),
```

Comment:
worthwhile to add an alias to `space_id` here just to avoid any potential isues in the future

ex - `space_id=dict(type='str', default='default', aliases=['space'])`